scriptmodules/emulators/dosbox-staging.sh

```diff
@@ -40,6 +40,9 @@ function build_dosbox-staging() {
     local meson_cmd="meson"
     [[ -f "$md_build/meson/meson.py" ]] && meson_cmd="python3 $md_build/meson/meson.py"
 
+    # disable speexdsp simd support on armv6 devices
+    isPlatform "arm6" && params+=(-Dspeexdsp:simd=false)
```

Comment:
If that isn't elegant, then I don't know what is!

RetroPie's shell setup system is one of the nicest out there. Great stuff, @joolswills and @cmitu :+1:

Reply:
Thank you. Shame about my typo in the first commit :-) (it's armv6 not arm6). Now fixed.